bin/test-build-on-all-distros

```diff
@@ -15,14 +15,24 @@ if [ -z "$1" ] || [ ! -d "$1" ]; then
   exit 1
 fi
 
-SOURCE_DIR="$1"
+SOURCE_DIR="$(realpath "$1")"
```

Comment:
This is not macos-compatible - this is similar to https://github.com/hhvm/homebrew-hhvm/pull/145, however we don't currently have a dependency list (or automated setup) for the packaging repo

Options seem to be:
- add an explicit check saying that realpath is required from *somewhere*
- if we start requiring non-standard packages on macos, we could also require the `pcregrep` or `grep` package (`grep` in homebrew is GNU grep, and adds a `ggrep` binary) - that said, to me the bash version you're doing here seems a little clearer than the old code so this might be desirable either way

Reply:
It's OK to not use `realpath` since I have dedicatedly avoided nested `cd` commands